clients/vscode/src/inline-edit/index.ts

```diff
@@ -51,44 +52,49 @@ export class InlineEditController {
     this.quickPick = quickPick;
   }
 
-  start() {
-    this.quickPick.show();
+  async start() {
+    this.args?.errorContext
+      ? await this.provideEditWithCommand(`Fix this error: ${this.args.errorContext}`)
```

Comment:
Please extract this into an standalone prompt template file to construct fix error prompt.